gitlab/gitlab.go

```diff
@@ -258,22 +264,20 @@ func (g *GitlabClient) UpdateJiraIntegration(p gitlab.Project, s ProjectJiraSett
 		return nil
 	}
 
-	j, err := g.GetJiraIntegration(p)
+	_, err := g.GetJiraIntegration(p)
```

Comment:
Wasn't this posting the same values before? Except now you're relying on gitlab to do so instead of intentionally doing so?

But, that's fine too. In that  case please remove this line and the error catch afterwards. It is no longer needed